### ISSUE TITLE
flitetranscoder binary is missing error message on mac os

### ISSUE THREAD

**Issue by justin808** [adobe-photoshop/generator-core#325]
https://forums.adobe.com/message/8054507#8054507

```
$ node app -f ../plugins
[debug:core 22:54:39.893 generator.js:105:17] Launching with config:
{}
[warning:core 22:54:42.843 generator.js:208:37] flitetranscoder binary is missing. flite functionality will not be available. { [Error: ENOENT, stat '/Applications/Adobe Photoshop CC 2015/Adobe Photoshop CC 2015.app/Contents/MacOS/flitetranscoder']
  errno: -2,
  code: 'ENOENT',
  path: '/Applications/Adobe Photoshop CC 2015/Adobe Photoshop CC 2015.app/Contents/MacOS/flitetranscoder' }
```

Any workaround?

Note, this file does not exist in the windows version.

**Comment by justin808** [comment]
If I run the command from http://tomkrcha.com/?p=3896

```
node app -f ../plugins
```

I get this output:

```
photoshop/generator-core (master u=) ✗ node app.js -file ../plugins                                                                                                                                                                           ✹ [20:42:35]
[debug:core 20:42:44.889 generator.js:105:17] Launching with config:
{}
[warning:core 20:42:55.261 generator.js:212:37] flitetranscoder binary is missing. flite functionality will not be available. { [Error: ENOENT: no such file or directory, stat '/Applications/Adobe Photoshop CC 2015/Adobe Photoshop CC 2015.app/Contents/MacOS/flitetranscoder']
  errno: -2,
  code: 'ENOENT',
  syscall: 'stat',
  path: '/Applications/Adobe Photoshop CC 2015/Adobe Photoshop CC 2015.app/Contents/MacOS/flitetranscoder' }
[info:core 20:42:55.342 generator.js:227:25] Detected Photoshop version: 16.0.1
Error processing plugin directory true
 [TypeError: Path must be a string. Received true]
Exiting with code -3: Generator failed to initialize: Generator requires at least one plugin to function, zero were loaded.
```

But if I go up a directory and run this command:

```
photoshop  node generator-core/ -f plugins                                                                                                                                                                                                      [20:43:40]
[debug:core 20:43:51.551 generator.js:105:17] Launching with config:
{}
[warning:core 20:43:51.713 generator.js:212:37] flitetranscoder binary is missing. flite functionality will not be available. { [Error: ENOENT: no such file or directory, stat '/Applications/Adobe Photoshop CC 2015/Adobe Photoshop CC 2015.app/Contents/MacOS/flitetranscoder']
  errno: -2,
  code: 'ENOENT',
  syscall: 'stat',
  path: '/Applications/Adobe Photoshop CC 2015/Adobe Photoshop CC 2015.app/Contents/MacOS/flitetranscoder' }
[info:core 20:43:51.754 generator.js:227:25] Detected Photoshop version: 16.0.1
[debug:core 20:43:51.761 generator.js:1755:21] Loading plugin: generator-getting-started
ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
init, config is {}
ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
```
(and lots more debug output).


Any information?

**Comment by justin808** [comment]
OK -- mysteries solved:

1. I don't get any error message on the branch: ps-16.0.0-release
2. I was not getting any plugins loaded when I used `-file` rather than `-f` Since Photoshop does not include the file flitetranscoder, maybe the general public should be using the ps-16.0.0-release branch?

**Comment by joelrbrandt** [comment]
@justin808 Sorry that this was a bit confusing. You came to the right conclusion: the flite transcoder is not in the 16.0 release of PS. That being said, the warning is harmless -- generator simply doesn't use that transcoder (and falls back to the old transcoder) if it is missing.

So, there's no need for the general public to use the ps-16.0.0-release branch instead of master. Of course, if they want to do their development against what shipped in 16.0, then using that branch is a good idea.

Closing bug.
